services-custom/s3-transfermanager/src/main/java/software/amazon/awssdk/custom/s3/transfer/internal/S3CrtUtils.java

```diff
@@ -48,6 +50,8 @@
 
 @SdkInternalApi
 public final class S3CrtUtils {
+    private static final String HEADER_USER_AGENT = "User-Agent";
+    private static final String USER_AGENT_STRING = "ft/s3-transfer";
```

Comment:
What does ft mean?

Reply:
`feature` per the SEP :)

Reply:
Ah, was looking at an old revision. Cool.